src/Types.jl

```diff
@@ -1136,9 +1136,8 @@ function update_registries(ctx::Context, regs::Vector{RegistrySpec} = collect_re
             # Using LibGit2.with here crashes julia when running the
             # tests for PkgDev wiht "Unreachable reached".
             # This seems to work around it.
-            local repo
+            repo = LibGit2.GitRepo(reg.path)
             try
-                repo = LibGit2.GitRepo(reg.path)
```

Comment:
AFICT this doesn't change the scope of `repo` just where it's assigned. I think the idea here was that the `GitRepo` operation could fail and we want to catch that.

Reply:
How about this then?

Reply:
yep, that's a better fix 👍